tools/auto_fill_inference_cache.py

```diff
@@ -134,8 +139,14 @@ def compile_and_cache_model(hf_model_id, batch_size, sequence_length, num_cores,
     # If a config file is provided, compile and cache all models in the file
     if args.config_file:
         logger.info(f"Compiling and caching models from config file: {args.config_file}")
-        with open(args.config_file, "r") as f:
-            config = json.load(f)
+        # check if conig file starts with https://
```

Comment:
You type too fast (me too) !
```suggestion
        # check if config file starts with https://
```